### DISCUSSION TITLE
Success on GT 330M Nvidia

### DISCUSSION THREAD

**Discussion by tahaklkt-osxinfo** [chris1111/Legacy-Video-patch#37] [Show and tell]
![Ekran Resmi 2021-04-04 13 11 59](https://user-images.githubusercontent.com/63608875/113505417-5834bd00-9547-11eb-835c-9ffa3cf5b0a9.png)

**Comment by chris1111** [comment]
thank you

**Reply by tahaklkt-osxinfo** [reply]
Do you add this to readme results ? Thanks!

**Reply by thinhdinh25** [reply]
> Do you add this to readme results ? Thanks!

Can you help me with nvcap patching
